Gulpfile.mjs

```diff
@@ -581,6 +594,41 @@ ${fs.readFileSync(path.join(path.dirname(input), "license"), "utf8")}*/
   );
 });
 
+gulp.task("build-cjs-bundles", () => {
+  if (!USE_ESM) {
+    fancyLog(chalk.yellow("Skipping CJS bundles because not compiling to ESM"));
```

Comment:
This could be quite confusing at first glance, but I can't come up with better phrases. Maybe ESM-compat CJS bundles?

Reply:
Sure :+1:

Reply:
```suggestion
    fancyLog(chalk.yellow("Skipping CJS-compat bundles for ESM-based builds, because not compiling to ESM"));
```
Does this look good?